### DISCUSSION TITLE
Broken link for the Windows GUI

### DISCUSSION THREAD

**Discussion by ec-ho-ra-mos** [MannLabs/alphapeptdeep#117] [General]
Hello Dr. Zeng,

I'm trying to download the Windows GUI, but it appears that the link is broken.

![image](https://github.com/MannLabs/alphapeptdeep/assets/61624696/8f8e7cbf-a832-40b2-9e0e-04a01a377d4d)

**Comment by jalew188** [comment]
Fixed in the new release

**Comment by liuwenrui917** [comment]
Hello Dr.Zeng
When I use the Windows GUI, it appears these errors.
<img width="1222" alt="4cf28bbb015d216fd1e12cb7dffdaaf" src="https://github.com/MannLabs/alphapeptdeep/assets/175176777/2825f598-3c4a-4a27-8776-7199a6ab4356">

**Comment by jalew188** [comment]
Hi @ec-ho-ra-mos As DIA-based transfer is only supported in AlphaDIA, so the PSM type list are changed. See #124

**Reply by liuwenrui917** [reply]
Hello Dr. Zeng
Thank you for your reply. And I want to ask you that I will use High-pH Reversed-Phase Fractionation to build a library with Fragpipe 22. But when I open pepdeep GUI，it appears an error like this. Do you know how to  solve this problem?
Thanks very much.
![fragpipe](https://github.com/user-attachments/assets/bd326640-5fc9-47eb-8b36-4ec2debb89ae)

**Comment by jalew188** [comment]
I see, this is due to the update of alphabase, I will fix this
